### DISCUSSION TITLE
New followers not fetched and updated on Y HYPE activity feed

### DISCUSSION THREAD

**Discussion by lulunac27a** [yhype/yhype-feedback#38] [Bug] [answered]
I get new followers on GitHub, but it's not updating the activity feed. The last activity feed was updated 3 days ago.

**Comment by antonkomarev** [comment]
@lulunac27a thanks for the report. It looks like something is broken. Followers removal is working, but adding is not. Will try to fix it ASAP.

**Reply by antonkomarev** [reply]
@lulunac27a I think I found an issue, need to wait for the next cycle of the sync.

**Reply by antonkomarev** [reply, marked as answer]
@lulunac27a I see that new followers are started to appear. Bug is fixed.

**Reply by lulunac27a** [reply]
It's working now
